tox.ini

```diff
@@ -38,7 +38,7 @@ deps =
 pip_pre = true
 
 commands =
-    pytest {posargs:-vv}
+    {posargs:pytest -vv}
```

Comment:
@ionelmc 
Was this done on purpose?  (or when resolving conflicts by accident)

Reply:
=> https://github.com/pytest-dev/pytest-cov/pull/255

Reply:
Oooof ... yes. I often do `tox -e something -- pip list -ol` and other things like that.

Reply:
I see.. but then you could also do `.tox/$TOXENV/bin/pip list -o`.